applications/portal/frontend/src/components/header/Header.tsx

```diff
@@ -101,22 +103,35 @@ interface HeaderProps {
  * @param {Object} props.user - The user object.
  */
 export const Header = ({
-                           login,
-                           logout,
-                           onExperimentCreation,
-                           user,
-                       }: HeaderProps) => {
+    login,
+    logout,
+    onExperimentCreation,
+    user,
+}: HeaderProps) => {
     const classes = useStyles();
-
+    const api = workspaceService.getApi();
     const match = useRouteMatch<RouteParams>(EXPERIMENTS_ROUTE);
     const experimentId = match ? match.params.id : null;
 
     const [modalKey, setModalKey] = React.useState(0)
     const [dialogOpen, setDialogOpen] = React.useState(false);
     const [experimentDialogOpen, setExperimentDialogOpen] = React.useState(false);
     const [errorMessage, setErrorMessage] = React.useState(undefined);
+    const [experiments, setExperiments] = React.useState([]);
+    const [fetchedExperiment, setFetchedExperiment] = React.useState<Experiment>(null);
     const menuAnchorRef = React.useRef(null);
 
+    const fetchExperiments = async () => {
```

Comment:
question: @Aiga115 do we still need this? Fetching all experiments? Imo this is obsolete